src/binding.cc

```diff
@@ -43,24 +43,23 @@ static const unsigned int kMaxLength = 0x3fffffff;
  * address, so there's a possibility of an inaccurate value being returned
  * on 64-bit systems.
  *
- * args[0] - Buffer - the Buffer instance get the memory address of
- * args[1] - Number - optional (0) - the offset of the Buffer start at
+ * info[0] - Buffer - the Buffer instance get the memory address of
+ * info[1] - Number - optional (0) - the offset of the Buffer start at
  */
 
 NAN_METHOD(Address) {
-  NanEscapableScope();
 
-  Local<Value> buf = args[0];
+  Local<Value> buf = info[0];
   if (!Buffer::HasInstance(buf)) {
-    return NanThrowTypeError("address: Buffer instance expected");
+    return Nan::ThrowTypeError("address: Buffer instance expected");
   }
 
-  int64_t offset = args[1]->IntegerValue();
+  int64_t offset = info[1]->IntegerValue();
```

Comment:
These should use `Nan::To<int64_t>(info[1]).FromJust()` or specify a default value and use the other unwrapping function.